core/src/test/java/com/opensymphony/xwork2/ognl/OgnlUtilTest.java

```diff
@@ -1284,12 +1295,11 @@ public void testAvoidCallingMethodsWithBraces() {
         assertEquals(expected.getMessage(), "Inappropriate OGNL expression: toString()");
     }
 
-    public void testAvoidCallingSomeClasses() {
+    public void testStaticMethodBlocked() {
         Foo foo = new Foo();
 
         Exception expected = null;
         try {
-            ognlUtil.setExcludedClasses(Runtime.class.getName());
```

Comment:
The exclusion list isn't checked here as it's already blocked by the static method check